Waking Shore Rares, Dragonrace Node

Ioney opened the pull request:
Rare quest ids might need to be changed/added. rewards added by personal loot and wowhead data.

Added Dragonrace Node Class
Only added one node for now.
Added a bluish silver to colors

Ioney committed:
Update templates.lua

The options panel wouldnt open if the map is in fullscreen/maximized.

Ioney committed:
Waking Shore Rares, Dragonrace Node

Rare quest id might need to be changed/added. rewards added by personal loot and wowhead data.

Added Dragonrace Node Class
Only added one node for now.
Added a bluish silver to colors

Ioney committed:
Revert "Update templates.lua"

This reverts commit ff1c39fa2358d9a4030283b978bd64a475fb9f5d.

Ioney committed:
Merge branch 'master' of https://github.com/Ioney/handynotes-plugins

Ioney committed:
Update waking_shores.lua

Ioney committed:
Locales

Forgot other Locales

Ioney committed:
Update common.lua

formatting

Dathwada commented:
all your added rares belong to bonus bosses as they're not related to any achievement.

Krovikan committed:
Updated class race DF esES

adavak committed:
Update zhCN.lua

adavak commented:
should be better locale "s" "second".

Lightuky committed:
Update frFR.lua - DF races

Dathwada committed:
Update dragon races

ntimes = normal times {bronze, silver, gold}
atimes = advanced times {bronze, silver, gold}

Krovikan committed:
Updated dragonraces DF esES

Ioney committed:
Dragonracing

Updated Dragonracing node formatting.
added Ohn'ahran Dragonracing nodes

Removed rares i added previously

Added currency change debug tracking

Ioney committed:
Dash Races

chagned formatting for dash races

Ioney committed:
formatting

Krovikan committed:
Updated dragonraces DF esES

zarillion committed:
Quests and currencies cannot share a check time

Ioney committed:
update
